sdk/nodejs/policy/policy.ts

```diff
@@ -13,37 +13,76 @@
 // limitations under the License.
 
 import * as pulumi from "@pulumi/pulumi";
-import { Resource } from "@pulumi/pulumi";
 import * as q from "@pulumi/pulumi/queryable";
+
 import { serve } from "./server";
 
+/**
+ * Arguments for constructing a new PolicyPack object.
+ */
 export interface PolicyPackArgs {
+    // Policies to associate with the policy pack.
     policies: Policy[];
 }
 
+/**
+ * Defines a collection of policies that will be used to validate resource state during a Pulumi
+ * stack preview or update.
+ */
 export class PolicyPack {
     private readonly policies: Policy[];
 
     constructor(private name: string, args: PolicyPackArgs) {
         this.policies = args.policies;
 
-        //
         // TODO: Wire up version information obtained from the service.
-        //
         const version = "1";
-
         serve(this.name, version, this.policies);
     }
 }
 
-/** A function that returns true if a resource definition violates some policy. */
+/**
+ * A Rule verifies the resource provided matches some expected state, and if not
+ * throws an exception of type AssertionError with a description of the problem.
+ * Otherwise, it is assumed the resource has been successfully verified.
+ *
+ * For example:
+ * ```
```

Comment:
Add a blank line above the code fence and specify a language, so it'll render correctly when we generate the API docs with proper syntax highlighting.

```suggestion

 * ```typescript
```